pysrc/dune_queries.py

```diff
@@ -0,0 +1,44 @@
+"""
+Localized account of all Queries related to this project's main functionality
```

Comment:
In my opinion similar to `dune/fetch/dune.py` a nice to have would be a todo here with an issue referenced to move this to the `dune-client` package.

Reply:
Will add. Also, I really hope that we don't have a a filepath like: `dune/fetch/dune.py` in this project.